.gitignore

```diff
@@ -37,3 +37,16 @@ _sketches.cpp
 
 # local override files
 Makefile.local
+
+# downloaded package sources
+pkg/cmsis-dsp/cmsis-dsp
+pkg/libcoap/libcoap
+pkg/libfixmath/libfixmath
+pkg/microcoap/microcoap
+pkg/micro-ecc/micro-ecc
+pkg/oonf_api/oonf_api
+pkg/openwsn/openwsn
+pkg/relic/relic
+pkg/tinydtls/tinydtls
+pkg/tlsf/tlsf
+pkg/wakaama/wakaama
```

Comment:
Again no strong opinion, but I would really dislike this PR being blocked by such a minor thing.